utils/crabotc.ts

```diff
@@ -92,3 +92,14 @@ export const validateBalance = (
 
   return { isValidOrder, response }
 }
+
+export const sortBidsForBidArray = (bids: Array<CrabOTCBid>, isSelling: boolean) => {
+  const sortedBids = bids.sort((a, b) => {
+    if (b.order.price === a.order.price) return Number(a.order.nonce) - Number(b.order.nonce)
```

Comment:
So when they are equal, you are giving preference to the order that came in first?

Reply:
Yes, will so this in the loom